cli/src/pcluster/config/cluster_config.py

```diff
@@ -1262,6 +1263,7 @@ def __init__(
         self.count = Resource.init_param(count, default=1)
         self.ssh = ssh
         self.iam = iam or LoginNodesIam(implied=True)
+        self.local_storage = local_storage or LocalStorage(implied=True)
```

Comment:
..and here we should initialize the value with a reasonable default.

Then when we initialize the pools we can pick the settings of the RootVolume of the HeadNode and create a LocalStorage object (with those settings) ensuring that `encrypted=true`.

Reply:
Done!